README.md

```diff
@@ -69,9 +85,21 @@ For XVF3510 devices these actions will be done as well:
 
    ```./setup.sh xvf3600-slave```
 
-   For XVF3610 devices, run the installation script as follows:
+   For XVF3610-UA devices, run the installation script as follows:
```

Comment:
When I first read the README file, it was not immediately clear that steps 3 and 4 were intended to take place on the Raspberry Pi.  That became clear when I read them again, but it may be worthwhile to include `One the Raspberry Pi,` at the beginning of item 3.